<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreWindow</class>
 <widget class="QWidget" name="StoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);
border:none;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>25</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>想要增加你的能力吗？</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>如果你有 25 个金币，你</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>可以任意选择一项：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>220</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>增加 800 点生命</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>增加 4 点攻击</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>增加 4 点防御</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>离开商店</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>125</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_old">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_red">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <zorder>graphicsView_3</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>graphicsView_old</zorder>
  <zorder>graphicsView_red</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
